filament_scale_enhanced/scale/__init__.py

```diff
@@ -0,0 +1,34 @@
+from typing import Dict
+from .scale import Scale
+
+SCALE_TYPES = ["rpi_hx711", "nau7802", "none"]
+
+
+# pylint: disable=import-outside-toplevel
+def get_scale(settings: Dict) -> Scale:
+    """Returns a fully configured Scale based on the settings provided"""
+
+    scale_type = settings["scale_type"]
+
+    if scale_type not in SCALE_TYPES:
+        raise ValueError(f"Invalid scale_type: {scale_type}")
+
+    if scale_type == "rpi_hx711":
+        from .rpi_hx711 import RPI_HX711
```

Comment:
I think I mentioned an importlib method previously. This keeps the logic out of the main entry point which is good.